### DISCUSSION TITLE
Create repository for `Dependencies`

### DISCUSSION THREAD

**Discussion by dankinsoid** [pointfreeco/swift-composable-architecture#1689] [General]
`Dependencies` works well outside of TCA, I found it very convenient to use `Dependencies` in backend Vapor projects (just had to remove `previewValue` value). 
Maybe it makes sense to move `Dependencies` to separated cross-platform repository?

**Comment by saroar** [comment]
hi how do you use it ? ping me on discord if it ok for you #1970 my id

**Reply by dankinsoid** [reply]
@saroar I cannot find you, I need your nickname

**Reply by saroar** [reply]
<img width="247" alt="Screenshot 2022-11-24 at 18 00 35" src="https://user-images.githubusercontent.com/8770772/203814790-8e399fe4-2de3-4151-ab9f-a1b5603807f2.png">
I hope it name `alif`

**Comment by stephencelis** [comment]
@dankinsoid We do plan to extract it, but curious what you mean by needing to remove `previewValue`? It should be totally fine to still support `previewValue`, so did it somehow get in the way for you?

**Reply by dankinsoid** [reply]
It's fine to support previewValue, I removed it in my project just because I don't need it and did not want problems when compiling on Linux.
Thanks for the answer and thanks for your libraries
